<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierClient</class>
 <widget class="QMainWindow" name="modifierClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1043</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1043</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion Client - Modifier Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../build-connectbdd-Desktop_Qt_5_9_6_MinGW_32bit-Debug/logo-domo.png</normaloff>../../build-connectbdd-Desktop_Qt_5_9_6_MinGW_32bit-Debug/logo-domo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="infoClient">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1011</width>
      <height>201</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="clientId">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Client ID</string>
    </property>
   </widget>
   <widget class="QTableView" name="infoMed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>1011</width>
      <height>221</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annuler">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Fermer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>-80</y>
      <width>981</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../ressource.qrc">:/new/prefix1/images/logo-domo3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>infoClient</zorder>
   <zorder>clientId</zorder>
   <zorder>infoMed</zorder>
   <zorder>modifier</zorder>
   <zorder>annuler</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../ressource.qrc"/>
 </resources>
 <connections/>
</ui>
